src/Model/DirectEventCapture.php

```diff
@@ -43,16 +43,10 @@ public function captureEvent(Event $event): void
         ];
 
         try {
-            $response = $this->fetch("POST", $this->dsn->getURL(), $body, $headers);
+            $this->fetch("POST", $this->dsn->getURL(), $body, $headers);
         } catch (RuntimeException $error) {
             error_log("SentryClient: unable to access Sentry service [{$error->getMessage()}]");
-
-            return; // NOTE: fail silently
         }
-
-        $data = json_decode($response, true);
-
-        $event->event_id = $data["id"];
```

Comment:
I think this might be an attempt to decouple the event capture from the client.

The client provides a UUID, so it should be redundant to reset it from the server response, but technically you could leave out the UUID to be backfilled from the server (https://develop.sentry.dev/sdk/event-payloads/types/ - see under "event_id").

So having this ensures that if the client calling `captureEvent()` hasn't provided a UUID for the event, that it is backfilled from the response.

It looks like the id is not used further if you just use the `DirectEventCapture` only, but making sure it is updated could be relevant if someone wishes to decorate the class with a decorator that might need to somehow subscribe to the event.

Reply:
Okay, so in order to avoid potentially breaking any code out in the wild that does this I've added the lines back in and included a fail-safe for handling json_decoding errors (which is what triggered the undefined index error in the tests)
https://github.com/kodus/sentry/pull/6/commits/61cfbf226a1ebf9dfccf03c0deb1f247d10faae2